### ISSUE TITLE
Scroll lock styles not being cleaned up on Navigate

### ISSUE THREAD

**Issue by Awesomolocity** [livewire/flux#839]
### Flux version

v1.0.29

### Livewire version

3.5.17

### What is the problem?

If you use `flux:menu` within `flux:dropdown`, it'll add `overflow:hidden` to the root HTML element.
This style attribute persists if you then `wire:navigate` away from the page, which is probably not desirable behavior.

You may say "Well obviously! It says to utilize `flux:navmenu` on the [dropdown documentation page](https://fluxui.dev/components/dropdown#navigation-menus) for this exactly reason". Which is true, however, is pretty easy to miss if you are copying your navbar directly from the [header documentation page](https://fluxui.dev/layouts/header), which uses this exact type of menu (it doesn't include `wire:navigate` - but also doesn't include any warning about the `overflow:hidden` behavior)

### Code snippets

```blade
<flux:dropdown position="top" align="start">
    <flux:profile name="Sheldon" avatar="https://ui-avatars.com/api/?name=Sheldon&background=random" />
    <flux:menu>
        <flux:menu.item icon="user" :href="route('profile')" wire:navigate>{{ __('Profile') }}</flux:menu.item>
        <flux:menu.item icon="arrow-right-start-on-rectangle" wire:click="logout">{{ __('Log Out') }}</flux:menu.item>
    </flux:menu>
</flux:dropdown>
```

Results in `overflow:hidden` persisting on root html element when navigating to Profile

### How do you expect it to work?

I'd either expect `overflow:hidden` to be removed from the element when navigating to a different page, OR for this possibly unexpected behavior to be more prominent within the documentation so that I could appropriately choose `flux:navmenu` as the sub-item and not experience this behavior.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by decantr** [comment]
I think the same issue is affecting links inside of a modal as `overflow:hidden` remains on the root element as observed here.

``` blade
<flux:modal.trigger name="example">
	<flux:button>Example</flux:button>
</flux:modal.trigger>

<flux:modal name="example">
	<flux:button href="example/route" wire:navigate>
		Example
	</flux:button>
</flux:modal>
```

**Comment by joshhanley** [comment]
@Awesomolocity and @decantr thanks for reporting!

I can confirm this is happening. At the moment the lockStroll styles are not being cleaned up on page navigate.

Below is a Volt component that demonstrates the issue. Make sure to have `npm run dev` running so the content has height.
- Scroll down to the "Launch modal" button and launch the modal
- Then click on the "Go to me" button in the modal, which should navigate you to the same page you are on
- Then try and scroll again and it will be locked

```blade
<?php

use Livewire\Volt\Component;

new class extends Component
{
    //
};
?>

<div>
    <div class="h-screen">Some long content to demonstrate the scroll not working</div>

    <flux:modal.trigger name="example">
        <flux:button>Launch modal</flux:button>
    </flux:modal.trigger>

    <flux:modal name="example">
        <flux:button :href="request()->url()" wire:navigate>
            Go to me
        </flux:button>
    </flux:modal>
    <div class="h-screen">Some morelong content to demonstrate the scroll not working</div>
</div>
```

**Comment by twoarmtom** [comment]
This sounds like it might be directly related to the issue that is also happening to me.  Building my application I'm encountering problems where I can not scroll to content when it is past the visible window.  After reading this issue, I can confirm that the behaviour I'm experiencing starts after I either use a flux:modal that uses the a redirect in the component with navigate: true after modal confirmation, or if I use a flux:dropdown to wire:navigate to another page.  

Any page I navigate to after using the dropdown or modal, that has content that extends beyond the visible browser window, I cannot scroll to it.

**Comment by FinnPaes** [comment]
Got the same issue here!

Using a flux dropdown like this:

```html
<flux:dropdown>
    <flux:button variant="ghost" size="sm" icon="ellipsis-horizontal" inset="top bottom"></flux:button>

    <flux:menu>
        <flux:menu.item icon="arrow-top-right-on-square" href="{{ route('uptime-details') }}" wire:navigate>@lang('Open details')</flux:menu.item>
    </flux:menu>
</flux:dropdown>
```

I made a temporary fix adding the following script to my app.blade.php (livewire layout) file:
```html
<script>
    document.addEventListener('livewire:navigated', () => {
        document.documentElement.removeAttribute('style');
    })
</script>
```

Of course this isn't a full solution to the problem because it completely removes the inline styling from the `<html>` tag. If you already have custom styling on the `<html>` tag it will also remove that.
For me that's not the case, so at least it's a working temporary fix!

Hopefully this will get properly fixed in a upcoming release!

**Comment by jeffchown** [comment]
@FinnPaes Have you tried removing only the `overflow` css property?, e.g.:

`document.documentElement.style.removeProperty('overflow');`

**Comment by FinnPaes** [comment]
@jeffchown Yes that would work as well, and might be safer temporarily solution!

**Comment by Awesomolocity** [comment]
- @FinnPaes If possible, in your dropdown, you can utilize `flux:navmenu` instead of `flux:menu` and that resolves the scroll issue. Might not work if you need menu-specific api tho.

**Comment by calebporzio** [comment]
Ah yep, this makes sense. I think `wire:navigate` should apply any differences on the HTML element that come in from a new request - @joshhanley wanna tackle that? thanks

**Comment by joshhanley** [comment]
I've submitted a PR https://github.com/livewire/livewire/pull/9149 to Livewire to add support for replacing html element attributes.

**Comment by joshhanley** [comment]
Livewire PR has been merged and released in Livewire v3.5.19.
